src/components/RequestModal/TvRequestModal.tsx

```diff
@@ -96,7 +99,7 @@ const TvRequestModal: React.FC<RequestModalProps> = ({
     selectedSeasons.length +
     (editRequest?.seasons ?? []).length;
 
-  const updateRequest = async () => {
+  const updateRequest = async (alsoAcceptRequest = false) => {
```

Comment:
```suggestion
  const updateRequest = async (alsoApproveRequest = false) => {
```
I think it's best to be consistent and use "approve" instead of "accept" throughout.

Reply:
Agree 👍